app/forms/api/profile_creation_form.rb

```diff
@@ -60,6 +60,8 @@ def deactivation_reason
         :gpo_verification_pending
       elsif pending_in_person_enrollment?
         :in_person_verification_pending
+      elsif threatmetrix_failed_and_needs_review?
+        :threatmetrix_review_pending
       end
```

Comment:
I'd like to register my vague discomfort at relying on this function to implicitly return `nil`. It works of course, and if it's a common pattern in the codebase I don't think it needs to be changed. Just a note that I consider it somewhat nonobvious.